src/apis/WikipediaApi.py

```diff
@@ -17,7 +17,7 @@
 class WikipediaApi(AWikiApi):
     def __init__(self):
         self.api_root = "https://en.wikipedia.org/api/rest_v1/"
-        self.headers = {'User-Agent': 'wong.mich@husky.neu.edu'}  # Wikipedia asks to provide contact info in user agent
+        self.headers = {'User-Agent': 'your@email.here'}  # Wikipedia asks to provide contact info in user agent
```

Comment:
This should really be an environment variable or something but whatever

Reply:
:man_shrugging: